civihr.install

```diff
@@ -641,3 +651,43 @@ function database_configuration_submit($form, &$form_state) {
   $install_state['settings_verified'] = TRUE;
   $install_state['completed_task'] = install_verify_completed_task();
 }
+
+/**
+ * Implements hook_form_alter to modify the install_configure_form and add
+ * options specific to the CiviHR installation
+ *
+ * @param array $form
+ * @param array $form_state
+ */
+function civihr_form_install_configure_form_alter(&$form, &$form_state) {
+  $form['demo_data'] = [
+    '#type' => 'fieldset',
+    '#title' => 'Demo Data',
+  ];
+
+  $form['demo_data']['create_demo_users'] = [
+    '#type' => 'checkbox',
+    '#title' => 'Create demo users',
+    '#description' => t('If selected, the system will create three users (civihr_staff, civihr_manager, and civihr_admin), so that you can try it out using different roles'),
+    '#default_value' => 1
+  ];
+
+  // When the custom submit function is added, the default one is lost and it is
+  // not called, so here we add first the default one and then out custom function
```

Comment:
There is a typo error in the latter part of this statement which should be `then our custom function`